core/src/test/java/org/mule/util/scan/ClasspathScannerTestCase.java

```diff
@@ -1,119 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.util.scan;
-
-import org.mule.tck.junit4.AbstractMuleTestCase;
-import org.mule.tck.testmodels.fruit.Apple;
-import org.mule.tck.testmodels.fruit.BloodOrange;
-import org.mule.tck.testmodels.fruit.Fruit;
-import org.mule.tck.testmodels.fruit.Orange;
-import org.mule.tck.testmodels.fruit.OrangeInterface;
-import org.mule.tck.testmodels.fruit.RedApple;
-import org.mule.util.scan.annotations.Marker;
-import org.mule.util.scan.annotations.Meta;
-import org.mule.util.scan.annotations.NonMeta;
-import org.mule.util.scan.annotations.SampleBeanWithAnnotations;
-import org.mule.util.scan.annotations.SampleClassWithAnnotations;
-
-import java.util.Set;
-
-import org.junit.Test;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-
-public class ClasspathScannerTestCase extends AbstractMuleTestCase
```

Comment:
in case we decide not to delete the tested class, remember to also recover this test